lib/include/jxl/decode.h

```diff
@@ -66,57 +66,58 @@ typedef enum {
  * @p size doesn't need to be a full image, only the beginning of the file.
  *
  * @return a flag indicating if a JPEG XL signature was found and what type.
- *   - JXL_SIG_NOT_ENOUGH_BYTES not enough bytes were passed to determine
- *       if a valid signature is there.
- *   - JXL_SIG_INVALID: no valid signature found for JPEG XL decoding.
- *   - JXL_SIG_CODESTREAM a valid JPEG XL codestream signature was found.
- *   - JXL_SIG_CONTAINER a valid JPEG XL container signature was found.
+ *   - @ref JXL_SIG_NOT_ENOUGH_BYTES if not enough bytes were passed to
+ * determine if a valid signature is there.
+ *   - @ref JXL_SIG_INVALID if no valid signature found for JPEG XL decoding.
+ *   - @ref JXL_SIG_CODESTREAM if a valid JPEG XL codestream signature was
+ * found.
```

Comment:
still a little bit of wonky indentation :)

Reply:
I think the extra space is necessary here because of the list items. Let me go scan the rest of it for wonky indentation though.

Reply:
Just TBC, I meant that's not enough indentation ;)